content/docs/main-concepts/api-clients.mdx

```diff
@@ -0,0 +1,67 @@
+
+The `DefaultContextUserName` property is for exactly that. It let's OrderCloud know what user context to provide when authentication occurs using the Client Credentials grant type. As far as OrderCloud knows, you *are* the default context user when using this grant type.
+
+Read more about supported OAuth workflows here.
+
+### Anonymous Shopping (Guest Checkout)
+Now that you understand how to enable the Client Credentials grant type you may think that this is all you would need in order to enable Anonymous Shopping - or what some call "Guest Checkout". **This is incorrect**, as anonymous shoppers should **not** be treated as if they *are* the default context user. They need to be treated as a temporary "copy" of the default context user.
+
+This is where the `IsAnonBuyer` property comes in. When enabled, many of the Me Perspective features are limited so that details about the default context user, like email address, cannot be altered. Additionally, the `access_token` will be encoded with an `orderid` which provides the anonymous user access to that order *only*.
+
+> It is important to note that adding a Client Secret is **not** necessary to enable Anonymous Shopping. In fact, it is **not recommended** to require a Client Secret for any client-side application.
+
+Read more about Anonymous Shopping here.
+
+### Token Durations
+Once a party has the appropriate level of access to your organization - it's important to control _how long_ that access will last. This is controlled by two properties: `AccessTokenDuration` and `RefreshTokenDuration`. The most important of the two is `AccessTokenDuration` - this is how long an `access_token` acquired using a specific API Client ID will live (in minutes).
+
+Once the primary `access_token` expires, if the API Client is configured with a _longer lasting_ `RefreshTokenDuration`, the original OAuth response will return a corresponding `refresh_token`. This can be used to aquire a fresh set of tokens and maintain user authentication in perpetuity. Long lasting token durations are not recommended for systems with a high level of data access in your organization as shorter `access_token` TTLs are more secure.
+
+## Use API Clients Liberally
+In general, we encourage OrderCloud creators to use API Clients as liberally as possible. Creating a new API Client for each system or isolated piece of functionality allows for quicker identification of bottlenecks in your solution. This is because every authenticated request will have the API Client that was used embedded in the `Authorization` header.
+
+Using API Clients liberally also adds to security. For example, if one system is compromised you can block access by deactivating its API Client or generating a new one without affecting availability of other systems in your solution.
+
+### Typical Use Cases for API Clients in an OrderCloud Solution
```

Comment:
I feel like this should get moved up before the examples are defined (i.e. before anonymous shopping). It felt kind of backwards to read about Anonymous Shopping and then see it listed as a "Typical Use Case"